File: ama/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_latlong/WoeidLatLong.java
package com.android.blawniczak.astroweather.domain.woeid_latlong;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WoeidLatLong {

    @SerializedName("query")
    @Expose
    public Query query;

}

File: paiwjs/AstroWeather2/app/src/main/java/com/android/blawniczak/astroweather/domain/woeid_latlong/WoeidLatLong.kt
package com.android.blawniczak.astroweather.domain.woeid_latlong

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WoeidLatLong {

    @SerializedName("query")
    @Expose
    var query: Query? = null

}
